app/utils/whitelabel.ts

```diff
@@ -1,31 +1,75 @@
+import AskDarcelImage from '../assets/img/askdarcel.svg';
+import BackgroundImage from '../assets/img/bg.png';
+import SearchByAlgoliaImage from '../assets/img/search-by-algolia.png';
+import SFFamiliesLogo from '../assets/img/sf-families.svg';
+import SFServiceLogo from '../assets/img/sf-service.svg';
+import SFSeal from '../assets/img/sf-seal.png';
 import config from '../config';
 
-/** Whether we should display the SF Service Guide branded version of the site */
-const isSFFamiliesSite = () => window.location.host.indexOf(config.SFFAMILIES_DOMAIN) > -1;
-
-const isSFServiceGuideSite = () => !isSFFamiliesSite()
-  && window.location.host.indexOf(config.MOHCD_DOMAIN) > -1;
-
-const getSiteTitle = () => {
-  if (isSFServiceGuideSite()) {
-    return 'SF Service Guide';
-  }
-  if (isSFFamiliesSite()) {
-    return 'SF Families';
-  }
-  return 'AskDarcel';
+// Use url to get key of config object
+// @ts-ignore
+const configKey = Object.keys(config).find((key) => config[key] === window.location.host) || 'default';
+// @ts-ignore
+let configurations: any;
```

Comment:
```suggestion
const configurations: any = {};
```

Hmm good point actually just tried this locally, and it errors because this object is undefined, causing a `Uncaught TypeError: Cannot set property 'title' of undefined` error

Reply:
When I fix that locally, everything looks OK to me. Can probably do some more thorough QA on staging after this is merged

Reply:
I would highly recommend that we don't give this a type of `any`, since that disables type checking on this object, which in turn disables type checking of the entire `whiteLabel` object being exported to other files.

I would suggest that we "transpose" the structure of this object so that the top-level properties are just the different site names (e.g. `SFFAMILIES_DOMAIN`, or in my other suggestion above, `SFFamilies`) and that all the other properties are nested under the site. This would allow us to define a type or interface that describes all of the properties that need to be set for each site, and it would allow the compiler to yell if we're missing any required properties.

The issue with the current structure is that if we added another site, it would be pretty easy to forget to add the new site key to each of the individual top-level properties.

An example of what I'm suggesting is the following:

```ts
interface WhiteLabelSite {
  title: string;
  siteUrl: string;
  appImages: {
    // Fill out each image here
  };
}

const configurations: Record<WhiteLabelSiteKey, WhiteLabelSite> = {};

configurations.SFFamilies = {
  title: 'SF Families',
  siteUrl: 'https://sffamilies.sfserviceguide.org/',
  appImages: { /* fill out images here */ }
}
```

This gives you type safety, preventing you from forgetting or typo'ing any properties, and it'll have better IDE hover information.